### PR TITLE
Reviewed Getting Started documentation

### PR DESCRIPTION
**Pull request by nathansimpson95**
FIxed spelling, grammar and broken link issues with the Getting Started documentation.

## Testing

 - [ ] List browser version(s) any admin UI changes were tested in:
 - [ ] Please confirm you've added (or verified) test coverage for this change.
 - [ ] Please confirm `npm run test-all` ran successfully.

### CODE REVIEW

File: docs/Getting Started/Setting-Up/part-1.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ At this point, we should have a `node_modules` directory and keystone should hav
 
 Create a new file, `keystone.js` and we're ready to start configuring keystone.
 
-Your `keystone.js` file is the launch file for keystone, which will connect keystone to your database, start both the database connection, and start your server running. This is where we will be adding configuration options to keystone as well, which allow us to change how keystone is running.
+Your `keystone.js` file is the launch file for keystone, which will connect keystone to your database, start the database connection, and start your server running. This is where we will be adding configuration options to keystone as well, which allow us to change how keystone is running.
```
Comment (stennie) [inline docs/Getting Started/Setting-Up/part-1.md]: Can we be consistent with Keystone uppercase?

File: docs/Getting Started/Setting-Up/part-1.md [hunk 2]
```diff
@@ -58,7 +58,11 @@ Keystone is ready on http://0.0.0.0:3000
 ------------------------------------------------
 ```
 
-Unfortunately all that's there is a 404 error page. We're going to solve that in [part 3](/getting-started/setting-up/part-3). In [Part 2](/getting-started/setting-up/part-2) we are going to focus on getting the database connected, and the admin UI up and running. You can do these two in either order.
+We should get a 404 page. That's ok! We're going to solve that in [part 3](/getting-started/setting-up/part-3). In [Part 2](/getting-started/setting-up/part-2) we are going to focus on getting the database connected, and the admin UI up and running. You can do these two in either order.
+
+## Next Steps
+Check out [part 2](/getting-started/setting-up/part-2) of our setting up keystone guide, which walks you through setting up your database, or if you want to read more about any of the parts we set up, you can check out these links:
```
Comment (stennie) [inline docs/Getting Started/Setting-Up/part-1.md]: Part 2: Database Setup

... "of this guide"

File: docs/Getting Started/Setting-Up/part-2.md [hunk 3]
```diff
@@ -120,8 +120,7 @@ email: { type: keystone.Field.Types.Email, unique: true },
 
 Email is similar to password in that it is using a keystone-specific field type, in this case to ensure that when this field is filled, it has the shape of an email. In addition, we have passed a second option of `unique: true`, which forces the field to be unique within the database. No doubling up on email addresses for accounts.
 
-// The following para really needs more work. Needs lightness and timing
-If you want to know about all the field types keystone offers, you can find the information find the full list of options in the [field docs](/api/field) Also, for the options like `unique` which are available to all fields, you can read more [here](/api/field/options), for when you are making your own models.
+If you want to know about all the field types keystone offers, you can find the full list of options in the [field docs](/api/field) Also, for the options like `unique` which are available to all fields, you can read more [here](/api/field/options), for when you are making your own models.
```
Comment (stennie) [inline docs/Getting Started/Setting-Up/part-2.md]: "field docs" => "Field API documentation"
"here" => "Field options API"

File: docs/Getting Started/Setting-Up/part-2.md [hunk 4]
```diff
@@ -234,11 +233,11 @@ Event.register();
 ## Next Steps
 Check out [part 3](/getting-started/setting-up/part-3) of our setting up keystone guide, which walks you through adding your own pages to your site, or if you want to read more about any of the parts we set up, you can check out these links:
 
-learn more about:
+## Learn more about:
```
Comment (stennie) [inline docs/Getting Started/Setting-Up/part-2.md]: Use page titles + proper case

File: docs/Getting Started/Setting-Up/part-3.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
 # Part 3: Routing
 
-Keystone is designed to do much of the setup of running an [express](expressjs.com) application out of your hands as well as allowing an easy configuration of the options.
+Keystone is designed to do much of the setup of running an [express](https://expressjs.com) application out of your hands as well as allowing an easy configuration of the options.
```
Comment (stennie) [inline docs/Getting Started/Setting-Up/part-3.md]: "express" => "ExpressJS"

File: docs/Getting Started/Setting-Up/part-4.md [hunk 6]
```diff
@@ -60,11 +60,11 @@ Your source tree should look like the following:
...
-## Adding a `addEvent` view
+## Adding an `addEvent` view
 
-First we want our pair of new files to make up the route and the view, these should be `routes/views/addEvent` and `templates/views/addEvent.pug`.
+First we want our pair of new files to make up the route and the view. These should be `routes/views/addEvent` and `templates/views/addEvent.pug`.
 
-As we are not a pug tutorial, here's a page we prepared earlier:
+As this is not a pug tutorial, here's a page we prepared earlier:
 
 ```jade
```
Comment (stennie) [inline docs/Getting Started/Setting-Up/part-4.md]: Is this GfM formatting still jade, or should it be `pug`?

File: docs/Getting Started/yo-generator.md [hunk 7]
```diff
@@ -134,19 +134,19 @@ For Keystone to provide authentication and session management to your applicatio
 - If you want your application to support session management, set the `session` option to true. Loading sessions incurs a small overhead, so if your application doesn't need sessions you can turn this off.
```
Comment (stennie) [inline docs/Getting Started/yo-generator.md]: Are there actually use cases for no sessions .. perhaps this is referring to persistent sessions versus in-memory?

File: docs/Getting Started/yo-generator.md [hunk 8]
```diff
@@ -134,19 +134,19 @@ For Keystone to provide authentication and session management to your applicatio
...
 - Sessions are persisted using an encrypted cookie storing the user's ID. Make sure you set the `cookie secret` option to a long, random string.
-- The user model must have a `canAccessKeystone` property (which can be a virtual method or a stored boolean) that says whether a user can access Keystone's Admin UI or not. \*Note\* If you choose to use a virtual method setting the value in mongodb directly will not authenticate correctly. A virtual method is useful when the criteria for access is more complex. See [Mongoose virtuals](mongoosejs.com/docs/guide.html#virtuals).
+- The user model must have a `canAccessKeystone` property (which can be a virtual method or a stored boolean) that says whether a user can access Keystone's Admin UI or not. \*Note\* If you choose to use a virtual method setting the value in mongodb directly will not authenticate correctly. A virtual method is useful when the criteria for access is more complex. See [Mongoose virtuals](http://mongoosejs.com/docs/guide.html#virtuals).
```
Comment (stennie) [inline docs/Getting Started/yo-generator.md]: This links to latest version of Mongoose docs (v5). Keystone is currently v4.

File: docs/Getting Started/yo-generator.md [hunk 9]
```diff
@@ -378,11 +378,11 @@ mixin flash-message(message, type)
 
 ### Using other template languages
 
-KeystoneJS supports any [template language supported by express](expressjs.com/en/api.html).
+KeystoneJS supports any [template language supported by express](https://expressjs.com/en/api.html).
```
Comment (stennie) [inline docs/Getting Started/yo-generator.md]: express => Express

File: docs/Getting Started/yo-generator.md [hunk 10]
```diff
@@ -378,11 +378,11 @@ mixin flash-message(message, type)
...
 
 Use the `view engine` option to specify the template language you want to use.
 
-If you want to use a custom template engine, set the `custom engine` option as well. For instance, [ejs](embeddedjs.com/) is supported by express by default, but you might want to use [ejs.locals](github.com/RandomEtc/ejs-locals) as a template engine in order to benefit from get extensions.
+If you want to use a custom template engine, set the `custom engine` option as well. For instance, [ejs](http://embeddedjs.com/) is supported by express by default, but you might want to use [ejs.locals](https://github.com/RandomEtc/ejs-locals) as a template engine in order to benefit from get extensions.
```
Comment (stennie) [inline docs/Getting Started/yo-generator.md]: Should this be EJS and http://ejs.co?

File: docs/Getting Started/yo-generator.md [hunk 11]
```diff
@@ -402,7 +402,7 @@ You'll want to add your own css, javascript, images and other files to your proj
 
 Keystone will serve any static assets you place in the public directory. This path is specified in `keystone.js` by the `static` option.
 
-It will also automatically generate `.css` or compressed `.min.css` files when a corresponding `.less` file is found in the public folder, as specified in `keystone.js` by the `less` option. For more information on LESS, see [lesscss.org](lesscss.org).
+It will also automatically generate `.css` or compressed `.min.css` files when a corresponding `.less` file is found in the public folder, as specified in `keystone.js` by the `less` option. For more information on LESS, see [lesscss.org](http://lesscss.org).
```
Comment (stennie) [inline docs/Getting Started/yo-generator.md]: "It" => ? (presumably Keystone)

File: docs/Getting Started/yo-generator.md [hunk 12]
```diff
@@ -414,7 +414,7 @@ When the option is set to `true`, Keystone will scan the `updates` directory for
 
 - next - the method to call when the update has finished running (including any internal callbacks)
 
-Updates are ordered using [Semantic Versioning](semver.org), and Keystone will only run them once (successfully executed updates are stored in your database, in a collection called `app_updates`).
+Updates are ordered using [Semantic Versioning](https://semver.org), and Keystone will only run them once (successfully executed updates are stored in your database, in a collection called `app_updates`).
```
Comment (stennie) [inline docs/Getting Started/yo-generator.md]: Could split into two sentences for clarity. I also think Keystone will only apply once, if successful.

File: docs/Getting Started/yo-generator.md [hunk 13]
```diff
@@ -449,14 +449,14 @@ Now you're ready to run your application, so execute the following in your proje
 
 Keystone will automatically apply the update, and then start a web server on the default port, 3000.
 
-To see your home page, point your browser at [localhost:3000](localhost:3000). You should see our **Hello World!** message.
+To see your home page, point your browser at [localhost:3000](http://localhost:3000). You should see our **Hello World!** message.
```
Comment (stennie) [inline docs/Getting Started/yo-generator.md]: our => a

File: docs/Getting Started/Setting-Up/part-4.md (hunk 6 above)
Comment (nathansimpson95) [reply docs/Getting Started/Setting-Up/part-4.md]: Doesn't look like it's supported yet

**Comment by stennie** [comment]
Thanks @nathansimpson95! Great first contribution :)

Cheers,
Stennie

**Pull request merged**